### DISCUSSION TITLE
How to Build Kivy from Source on Windows?

### DISCUSSION THREAD

**Discussion by RicardoTactiql** [kivy/kivy-github-discussions#2] [Q&A]
Hello, I am using mingw64 on windows and I am looking to use Kivy. When I try to pip install Kivy directly i get  ```No matching distribution found for kivy_deps.gstreamer_dev~=0.3.3```. More context here: https://github.com/kivy/kivy/issues/8412

I am trying to build Kivy on Windows, but when I look at the instructions there is no script for Windows, only for Linux and Mac: https://kivy.org/doc/stable/gettingstarted/installation.html#from-source

How can I build Kivy from source on Windows?

Thanks!

**Comment by Julian-O** [comment]
I do not believe Mingw64 is supported. The instructions on [installing Kivy from source on Windows](https://kivy.org/doc/stable/installation/installation-windows.html#install-source-win) specifies the compiler you will need.

If you install with mingw, you are probably on your own.
